code/game/objects/structures/flora.dm

```diff
@@ -7,6 +7,16 @@
 	layer = ABOVE_HUMAN_LAYER
 	var/protects_against_weather = FALSE
 
+/obj/structure/flora/tree/attackby(var/obj/item/I, mob/user)
+	if(I.sharp && I.edge && (I.force > 1) )
+		visible_message(SPAN_NOTICE("\The [user] starts chopping at \the [src] with \a [I]."), SPAN_NOTICE("You begin chopping \the [src]."))
+		if(do_after(user, 5 SECONDS))
+			visible_message(SPAN_NOTICE("\The [user] fell \the [src]!"), SPAN_NOTICE("You fell \the [src]."))
```

Comment:
Maybe make this drop wood.

Reply:
I'll probably try calling dismantle() instead of qdel here then. Since its a structure, it'll drop the mats its made of and we can drop wood that way.

Reply:
done.